jormungandr-integration-tests/src/common/jcli_wrapper/certificate/commands.rs

```diff
@@ -26,6 +26,9 @@ impl CertificateCommands {
             .arg("new")
             .arg("stake-delegation")
             .arg(&delegation_id)
+            // This was added as part of updating structopt to 0.3+, it changed the interface
+            // and required to add `--` before the stake pool ids
+            .arg("--")
```

Comment:
I had to add this in order to make the command work. I could not find the change in the documentation of `structopt` but it is due to a change in it. It means that now a command should be run with:
```
jcli.exe certificate new stake-delegation delegation_id -- stake_pool_id
```
I'm taking a look to see if I can make it work without the `--` to keep the same behavior as in previous versions

Reply:
is that maybe a bug in structopt ?

Reply:
Strange. I tested on macOS with and without `--` and both work just fine.

Reply:
@eugene-babichenko , did you check with this version (`0.3^`)?

Reply:
I made a `cargo clean`, `cargo build -all` and `cargo test`, it is working as expected. I may had a bad build when testing it before.

Reply:
Great to hear. Yes, I was playing with this branch.